clients/admin-ui/cypress/e2e/action-center/assets-results.cy.ts

```diff
@@ -696,4 +696,115 @@ describe("Action center Asset Results", () => {
       });
     });
   });
+
+  describe("Adding and removing categories of consent from results", () => {
+    const webMonitorKey = "my_web_monitor_1";
+    const systemId = "system_key-8fe42cdb-af2e-4b9e-9b38-f75673180b88";
+    const rowUrns = [
+      "my_web_monitor_1.GET.td.doubleclick.net.https://td.doubleclick.net/td/rul/11020051272",
+      "my_web_monitor_1.POST.www.google.com.https://www.google.com/ccm/collect",
+      "my_web_monitor_1.POST.www.google.com.https://www.google.com/ccm/anchor",
+      "my_web_monitor_1.GET.td.doubleclick.net.https://td.doubleclick.net/td/rul/697301175",
+    ];
+
+    beforeEach(() => {
+      cy.login();
+      stubPlus(true);
+      cy.intercept("GET", "/api/v1/plus/discovery-monitor/*/results*", {
+        fixture:
+          "detection-discovery/activity-center/system-asset-results-with-consent",
+      }).as("getSystemAssetResultsWithConsent");
+      cy.intercept("PATCH", "/api/v1/plus/discovery-monitor/*/results", {
+        response: 200,
+      }).as("patchAssets");
+
+      cy.visit(`${ACTION_CENTER_ROUTE}/${webMonitorKey}/${systemId}`);
+      cy.wait("@getSystemAssetResultsWithConsent");
+    });
+
+    it("should show preferred_data_uses as categories of consent", () => {
+      // Rows with preferred_data_uses should display their consent category
+      cy.getAntTableRow(rowUrns[2]).within(() => {
+        cy.contains("analytics").should("exist");
+      });
+      cy.getAntTableRow(rowUrns[3]).within(() => {
+        cy.contains("marketing.advertising.first_party.targeted").should(
+          "exist",
+        );
+        cy.contains("analytics").should("exist");
```

Comment:
these are also chain-able
```suggestion
        cy.getAntCellWithinRow(3)
          .should("contain", "marketing.advertising.first_party.targeted")
          .should("contain", "analytics");
```